config/rbac/role.yaml

```diff
@@ -58,7 +64,13 @@ rules:
   - cephobjectstoreusers
   - cephrbdmirrors
   verbs:
-  - '*'
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+   - watch
```

Comment:
`- watch` looks indented an extra space in many places

Reply:
+1